AdvancedRocketry/blocks/index.php

```diff
@@ -1,18 +1,19 @@
 <?php
```

Comment:
The changes here are just to correct a few names to match the in-game names, and to alphabetize things which were out of order.